index.js

```diff
@@ -48,6 +76,9 @@ var TitleBar = function(options) {
 util.inherits(TitleBar, events.EventEmitter);
 
 TitleBar.prototype.appendTo = function(target) {
+	if (this._options.os == 'mac') var style = styleMac;
+	else if (this._options.os == 'win') var style = styleWin;
```

Comment:
Looks kind of weird. It would be better to declare `var style` outside the if statement.